goldmane/proto/methods.go

```diff
@@ -66,9 +96,9 @@ func makeNamePart(h *PolicyHit) string {
 	case PolicyKind_StagedKubernetesNetworkPolicy:
 		namePart = fmt.Sprintf("%s/staged:knp.default.%s", h.Namespace, h.Name)
 	case PolicyKind_StagedGlobalNetworkPolicy:
-		namePart = fmt.Sprintf("staged:%s.%s", h.Tier, h.Name)
+		namePart = fmt.Sprintf("%s.staged:%s", h.Tier, h.Name)
 	case PolicyKind_StagedNetworkPolicy:
-		namePart = fmt.Sprintf("%s/staged:%s.%s", h.Namespace, h.Tier, h.Name)
+		namePart = fmt.Sprintf("%s/%s.staged:%s", h.Namespace, h.Tier, h.Name)
 	case PolicyKind_AdminNetworkPolicy:
 		namePart = fmt.Sprintf("kanp.adminnetworkpolicy.%s", h.Name)
```

Comment:
Even though it might be less readable, I'm in favor of utilizing constants for `knp`, kanp`, `baselineadminnetworkpolicy` etc.. Might help avoid misspelling these if we were to add any other checks down the line

Reply:
Yeah... might just be personal preference but I hope that the unit tests will prevent that sort of bug from showing up.

The constants make it so much less readable that I find it easier to write bugs into the code that way. I'll see if I can do it cleanly though....